src/components/ClosePaneButton.test.js

```diff
@@ -12,16 +12,15 @@ describe('<ClosePaneButton />', () => {
     expect(wrapper.find('button').length).toEqual(1);
   });
 
-  it('removes element with matching id', () => {
-    const mockRef = {
-      current: '<div id="parent-container">',
-    };
+  it('calls on the parent removeGridComponent', () => {
+    let calledMockFlag = false;
+    const mockRemoveGridComponent = () => { calledMockFlag = !calledMockFlag; return null; };
```

Comment:
- Would it be safer to just say `calledMockFlag = true`? I'm worrying about a case where it's getting called twice.
- Is the `return null` necessary?

Reply:
1) For this test, it should only be called once. 
2) Return null is not necessary. I think lint was yelling at me for not returning something and I forgot to investigate further.